bindings/python/tests/test_encoding.py

```diff
@@ -0,0 +1,24 @@
+# type: ignore
+from slimt.utils import toJSON
+from slimt import Encoding
+
+
+def test_basic(service, models, sample):
+    source = "no sé"
+    model = models[1]
+    responses_byte = service.translate(
+        model, [source], html=False, encoding=Encoding.Byte
+    )
+    responses_utf8 = service.translate(
+        model, [source], html=False, encoding=Encoding.UTF8
+    )
+    for response in responses_byte:
+        print(dir(response))
+        print(toJSON(response, indent=4))
+        extracted_byte = response.source.word(0, 0)
+    for response in responses_utf8:
+        print(dir(response))
+        print(toJSON(response, indent=4))
+        range = response.source.word_as_range(0, 0)
+        extracted_utf8 = response.source.text[range.begin : range.end]
+    assert extracted_utf8 == extracted_byte
```

Comment:
Check for all segments. I was thinking of using python for the gold truth, instead of our other implementation.

```py
from collections import namedtuple

Pair = namedtuple("Pair", 'byte utf8')

text = Pair(
    byte = utf8.text.encode(), # encode to get bytes, via Python (gold)
    utf8 = utf8.text
)

for sentence_id, word_id in generate(...):
  range = Pair(
    byte = byte.word_as_range(sentence_id, word_id),
    utf8 = utf8.word_as_range(sentence_id, word_id)
  )
  
  expected = text.utf8[range.utf8.begin:range.utf8.end]
  
  # Slice "byte" range using info, and encode back to utf-8 to compare. 
  reconstructed = text.byte[range.byte.begin:range.byte.end].encode("utf-8")
  
  assert(expected == reconstructed)

```